features.json

```diff
@@ -1,150 +1,204 @@
-            "label": "Enable BrowZine Web Journal Issue Links",
-            "options": [
-              {
-                "name": "true",
-                "value": true
-              },
-              {
-                "name": "false",
-                "value": false
-              }
-            ]
-          }
-        },
-        {
-          "key": "journalBrowZineWebLinkText",
-          "type": "input",
-          "defaultValue": "View Journal Contents",
-          "templateOptions": {
-            "label": "BrowZine Web Journal Issue Link Text"
-          }
-        },
-        {
-          "key": "articleBrowZineWebLinkTextEnabled",
-          "type": "select",
-          "defaultValue": true,
-          "templateOptions": {
-            "label": "Enable BrowZine Web Article In Context Links",
-            "options": [
-              {
-                "name": "true",
-                "value": true
-              },
-              {
-                "name": "false",
-                "value": false
-              }
-            ]
-          }
-        },
-        {
-          "key": "articleBrowZineWebLinkText",
-          "type": "input",
-          "defaultValue": "View Issue Contents",
-          "templateOptions": {
-            "label": "BrowZine Web Article In Context Link Text"
-          }
-        },
-        {
-          "key": "articlePDFDownloadLinkEnabled",
-          "type": "select",
-          "defaultValue": true,
-          "templateOptions": {
-            "label": "Enable BrowZine Web PDF Download Links",
-            "options": [
-              {
-                "name": "true",
-                "value": true
-              },
-              {
-                "name": "false",
-                "value": false
-              }
-            ]
-          }
-        },
-        {
-          "key": "articlePDFDownloadLinkText",
-          "type": "input",
-          "defaultValue": "Download PDF",
-          "templateOptions": {
-            "label": "BrowZine Web PDF Download Link Text"
-          }
-        },
-        {
-          "key": "printRecordsIntegrationEnabled",
-          "type": "select",
-          "defaultValue": true,
-          "templateOptions": {
-            "label": "Enable Print Records Integration",
-            "options": [
-              {
-                "name": "true",
-                "value": true
-              },
-              {
-                "name": "false",
-                "value": false
-              }
-            ]
-          }
-        }
-      ]
-    }
-  },
+		"face": "https://avatars2.githubusercontent.com/u/11337842?s=200&v=4",
+		"notes": "Add 'Open Access available via unpaywall' link to search-result-avaliability-line-after in Primo New UI",
+		"who": "BU Libraries",
+		"what": "bulib-unpaywall",
+		"linkGit": "https://github.com/bulib/primo-explore-bu/tree/master/packages/unpaywall",
+		"npmid": "primo-explore-unpaywall",
+		"version": "1.0.4",
```

Comment:
please update this to `"1.0.5"`. I've published a new version of this package that I hope solves this issue.

Reply:
code changes are visible in `bulib/primo-explore-bu` [commit `7f515f3`](https://github.com/bulib/primo-explore-bu/commit/7f515f3f164cb64ee438945e46df57dbfdc64aac)